source/Octopus.Client/Model/VersionControl/ConvertProjectToVersionControlledCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Octopus.Client.Model.VersionControl
+{
+    public class ConvertProjectToVersionControlledCommand
```

Comment:
As discussed with @tothegills and @ankithkonda: this makes the intention of the API call clear - it's an action, not a resource. It's still hidden behind the `Beta()` interface to the `ProjectsRepository` so we can change it any time we like, but it feels like a good place to start.